### PR TITLE
Merge `main` to `prerelease` for v1.0.0

### PR DESCRIPTION
**Pull request by mcumming**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/xcsync/pull/175)

### CODE REVIEW

**Review by haritha-mohan — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

File: src/xcsync/Resources/Strings.resx [hunk 1]
```diff
@@ -160,14 +160,18 @@
 		<value>used after the generate command to synchronize changes from the generated Xcode project back to the .NET project</value>
 	</data>
 	<data name="Commands.Watch.Description" xml:space="preserve">
-		<value>syncs changes between the .NET project and Xcode project automatically</value>
+		<value>generates a Xcode project, then continuously synchronizes changes between the Xcode project and the .NET project</value>
```
Comment (stephen-hawley) [inline src/xcsync/Resources/Strings.resx:163]: very minor: "a Xcode" -> "an Xcode"

File: src/xcsync/Resources/Strings.resx [hunk 2]
```diff
@@ -276,9 +284,24 @@
 
 	<!-- Watch Messages -->
 	<data name="Watch.HeaderInformation" xml:space="preserve">
-		<value>Syncing changes between project '{0}' and '{1}'.</value>
-		<comment>{0} is the path to the project file, {1} is the path to the target project</comment>
+		<value>Continuously syncing files between '{0}' and '{1}' for '{2}' platform.</value>
+		<comment>{0} is the path to the project file, {1} is the path to the target project, {2} is the target platform moniker</comment>
+	</data>
+	<data name="Watch.ReceivedCtrlC" xml:space="preserve">
+		<value>Received CTRL-C, exiting.</value>
```
Comment (stephen-hawley) [inline src/xcsync/Resources/Strings.resx:291]: Capitalization of `CTRL` should be consistent with the string in line 303. I don't think it matters which way, as long as they match.

File: src/xcsync/Resources/xlf/Strings.de.xlf [hunk 3]
```diff
@@ -53,10 +78,15 @@
         <note />
       </trans-unit>
       <trans-unit id="Commands.Watch.Description">
-        <source>syncs changes between the .NET project and Xcode project automatically</source>
-        <target state="translated">synchronisiert Änderungen zwischen dem .NET-Projekt und dem Xcode-Projekt automatisch</target>
+        <source>generates a Xcode project, then continuously synchronizes changes between the Xcode project and the .NET project</source>
```
Comment (stephen-hawley) [inline src/xcsync/Resources/xlf/Strings.de.xlf:81]: a Xcode -> an Xcode

File: src/xcsync/Resources/xlf/Strings.de.xlf [hunk 4]
```diff
@@ -53,10 +78,15 @@
...
+        <target state="new">generates a Xcode project, then continuously synchronizes changes between the Xcode project and the .NET project</target>
```
Comment (stephen-hawley) [inline src/xcsync/Resources/xlf/Strings.de.xlf:82]: same

File: src/xcsync/Resources/xlf/Strings.fr.xlf [hunk 5]
```diff
@@ -53,10 +78,15 @@
         <note />
       </trans-unit>
       <trans-unit id="Commands.Watch.Description">
-        <source>syncs changes between the .NET project and Xcode project automatically</source>
-        <target state="translated">synchronise automatiquement les modifications entre le projet .NET et le projet Xcode</target>
+        <source>generates a Xcode project, then continuously synchronizes changes between the Xcode project and the .NET project</source>
```
Comment (stephen-hawley) [inline src/xcsync/Resources/xlf/Strings.fr.xlf:81]: Same

File: test/xcsync.e2e.tests/usecases/GenerateThenSyncWithNoChangesTests.cs [hunk 6]
```diff
@@ -101,7 +101,7 @@ public async Task GenerateThenSync_WhenPwdIsProjectDir_DoesNotThrow (string proj
 	}
 
 
-	[Theory]
+	[Theory (Skip = "Needs work")]
```
Comment (stephen-hawley) [inline test/xcsync.e2e.tests/usecases/GenerateThenSyncWithNoChangesTests.cs:104]: Is there an issue for this?

**Review by stephen-hawley — APPROVED** [review]
Some minor changes needed for grammar, but if you want to wait until the next time it needs translation, just open an issue and get to it later.

Like seeing all the tests.

**Pull request merged**